src/transformers/models/sam/modeling_sam.py

```diff
@@ -856,11 +900,115 @@ def forward(self, hidden_states: torch.Tensor, output_attentions=False) -> torch
         return outputs
 
 
+class SamSdpaVisionAttention(SamVisionAttention):
+    """
+    Multi-head Attention block with relative position embeddings.
+    Using SDPA instead of the default attention.
+    """
+
+    def __init__(self, config, window_size):
+        super().__init__(config, window_size)
+
+    def add_decomposed_rel_pos(
```

Comment:
this should be the same as the parrent one no?

Reply:
Oh nice then let's add a small comment about that!

Reply:
Added a small comment about it in d486e627c64c6982ada52805de75857c86ad8c74